Tests/iaas/flavor-naming/flavor_names.py

```diff
@@ -192,6 +192,11 @@ class CPUBrand:
     })
     perf = TblAttr("Performance", {"": "Std Perf", "h": "High Perf", "hh": "Very High Perf", "hhh": "Very Very High Perf"})
 
+    def __init__(self, cpuvendor="i", cpugen=0, perf=""):
```

Comment:
It's a bit strange that this is the only class of the bunch that has a constructor, which might raise some eyebrow, and it is only used at one occurrence, but I guess these points aren't sufficient to warrant any real objection.

Reply:
The fact we don't even need constructors for most is the exception I think, not that we have one here.
The c'tor does what I would expect from a c'tor, so defining it and using it makes the code where the logic happens more readable IMVHO.